README.md

```diff
@@ -67,37 +67,44 @@ The example app is a catalogue of the core components provided by this library.
 
 4. Create an environment file
 
-   Create a file at `examples/discovery-search-app/.env.local` file, and populate the following values from your Discovery project:
+   Copy the `examples/discovery-search-app/.env` file to `examples/discovery-search-app/.env.local` file, and populate the following values from your Discovery project:
 
    ```
-   REACT_APP_PROJECT_ID=<project_id to query>
-   CLUSTER_USERNAME=<cluster username>
-   CLUSTER_PASSWORD=<cluster password>
-   CLUSTER_PORT=<cluster port>
-   CLUSTER_HOST=<cluster hostname>
+   REACT_APP_PROJECT_ID={REPLACE_ME}
+   CLUSTER_USERNAME={REPLACE_ME}
+   CLUSTER_PASSWORD={REPLACE_ME}
+   CLUSTER_PORT={REPLACE_ME}
+   CLUSTER_HOST={REPLACE_ME}
    ```
 
-   1. `REACT_APP_PROJECT_ID` is contained in the URL when viewing your Discovery project on the CP4D cluster (ex. `https://{CLUSTER_HOST}:{CLUSTER_PORT}/discovery/{RELEASE_NAME}/projects/{REACT_APP_PROJECT_ID}/workspace`)
-   2. `CLUSTER_USERNAME` the username used to log in to your CP4D dashboard and access your instance of Discovery
-   3. `CLUSTER_PASSWORD` the password used to log in to your CP4D dashboard and access your instance of Discovery
+   1. `REACT_APP_PROJECT_ID` is a guid contained in the URL (sample URL: `https://{CLUSTER_HOST}:{CLUSTER_PORT}/discovery/{RELEASE_NAME}/projects/{REACT_APP_PROJECT_ID}/workspace`) when viewing your Discovery project on the CP4D cluster (ex. `97ba736d-6563-4270-a489-c19d682b6369`)
+   2. `CLUSTER_USERNAME` the username used to log in to your CP4D dashboard and access your instance of Discovery (ex. `my_cp4d_username`)
+   3. `CLUSTER_PASSWORD` the password used to log in to your CP4D dashboard and access your instance of Discovery (ex. `my_cp4d_password`)
    4. `CLUSTER_PORT` defaults to `443` unless configured otherwise
-   5. `CLUSTER_HOST` the base URL of your CP4D cluster (ex. `{}.com`)
-
-For more information about configuring your Cloud Pak for Data cluster, see https://www.ibm.com/support/producthub/icpdata/docs/content/SSQNUZ_current/cpd/overview/overview.html
+   5. `CLUSTER_HOST` the base URL of your CP4D cluster (ex. `example.com` - no protocol/port)
 
 5. Build the React components:
 
    ```
    yarn workspace @ibm-watson/discovery-react-components run build
    ```
 
-6. In one terminal, start the server:
+6. Perform one of the two steps
 
-   ```
-   yarn workspace discovery-search-app run server
-   ```
+   - Run the setup script
+     ```
+     yarn workspace discovery-search-app run server:setup
```

Comment:
Is it worth mentioning this essentially does what you're telling them to do below?